src/library/library.cpp

```diff
@@ -105,6 +107,9 @@ Library::Library(
 
     m_pCrateFeature = new CrateFeature(this, m_pConfig);
     addFeature(m_pCrateFeature);
+
+    m_pRelationsFeature = new RelationsFeature(this, m_pConfig);
+    addFeature(m_pRelationsFeature);
```

Comment:
move this below the crate connections (assuming it's supposed to appear below Crates)